src/OrchardCore/OrchardCore.ContentManagement.GraphQL/ServiceCollectionExtensions.cs

```diff
@@ -25,9 +27,17 @@ public static IServiceCollection AddContentGraphQL(this IServiceCollection servi
 
             services.AddOptions<GraphQLContentOptions>();
 
+            services.AddSingleton(typeof(IIndexPropertyProvider), typeof(IndexPropertyProvider<ContentItemIndex>));
```

Comment:
services.AddSingleton<IIndexPropertyProvider, IndexPropertyProvider<ContentItemIndex>>()

Reply:
oops think that was left over in that style from when the interface was generic too